pandas/core/internals.py

```diff
@@ -5673,6 +5699,8 @@ def is_na(self):
             if not values._null_fill_value and values.sp_index.ngaps > 0:
                 return False
             values_flat = values.ravel(order='K')
+        elif isinstance(self.block, ExtensionBlock):
```

Comment:
so you should add a ``.ravel()`` method to EABlock (maybe Array), which just returns self.

Reply:
Yeah, I thought about that. Seems inelegant, but reasonable.

Reply:
Ahh, hmm. Categorical.ravel actually casts to an ndarray. I don't think we want that though... And we don't want to be inconsistent between Catgorical and other EAs.

Reply:
If possible, I rather prefer to avoid adding `.ravel()` to the ExtensionArray interface (it's not useful as long as they are limited to 1D, and it would indeed be unclear what the expected return value would be (and EA or ndarray))